modules/mirror-registry-introduction.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-introduction_{context}"]
+== Mirror registry for Red Hat Openshift introduction
+
+For disconnected deployments of {product-title}, a container registry is required to carry out the installation of the clusters. If users want to run a production-grade registry service on such a cluster, a separate registry deployment is required to install the first cluster. The _mirror registry for Red Hat Openshift_ addresses this need and is included in every OpenShift subscription. It is available for download on the link:https://catalog.redhat.com/software/containers/openshift/mirror-registry-rhel8/61a52d4fc17162a20c1c6863[mirror registry for Red Hat Openshift] page.
+
+The mirror registry for Red Hat Openshift allows users to install a small-scale version of Red Hat Quay and its required components using the `mirror-registry` command line interface (CLI) tool. The mirror registry for Red Hat Openshift is deployed automatically with pre-configured local storage and a local database. It also includes auto-generated user credentials and access permissions with a single set of inputs and no additional configuration choices to get started.
+
+The mirror registry for Red Hat Openshift provides a pre-determined network configuration and reports deployed component credentials and access URLs upon success. A limited set of optional configuration inputs like fully qualified domain name (FQDN) services, superuser name and password, and custom TLS certificates are also provided. This provides users with a container registry so that they can easily create an offline mirror of all {product-title} release content when running {product-title} in restricted network environments.
+
+The mirror registry for Red Hat Openshift is limited to hosting images that are required to install a disconnected {product-title} cluster, such as Release images or Red Hat Operator images. It uses local storage on your {op-system-base-full} machine, and storage supported by {op-system-base} is supported by the mirror registry for Red Hat Openshift. Content built by customers should not be hosted by the mirror registry for Red Hat Openshift.
+
+Unlike Red Hat Quay, the mirror registry for Red Hat Openshift is not a highly-available registry and only local filesystem storage is supported. Although it is technically possible, using the mirror registry for Red Hat Openshift with more than one cluster is discouraged, because multiple clusters can create a single point of failure when updating your cluster fleet. It is advised to leverage the mirror registry for Red Hat Openshift to install a cluster that can host a production-grade, highly-available registry such as Red Hat Quay, which can serve {product-title} content to other clusters.
```

Comment:
```suggestion
Unlike Red Hat Quay, the mirror registry for Red Hat Openshift is not a highly-available registry and only local file system storage is supported. Using the mirror registry for Red Hat Openshift with more than one cluster is discouraged, because multiple clusters can create a single point of failure when updating your cluster fleet. It is advised to leverage the mirror registry for Red Hat Openshift to install a cluster that can host a production-grade, highly-available registry such as Red Hat Quay, which can serve {product-title} content to other clusters.
```